src/components/skills-quiz/SearchProgramCard.jsx

```diff
@@ -145,6 +144,116 @@ const SearchProgramCard = ({ index }) => {
     [JSON.stringify(programs)],
   );
 
+  const loadingCard = () => (
+    <Card>
+      <Card.ImageCap
+        as={Skeleton}
+        duration={0}
+      />
+
+      <Card.Header
+        title={
+          <Skeleton duration={0} data-testid="program-title-loading" />
+        }
+      />
+
+      <Card.Section>
+        <Skeleton duration={0} data-testid="program-type-loading" />
+      </Card.Section>
+
+      <Card.Section>
+        <Skeleton duration={0} data-testid="partner-key-loading" />
+      </Card.Section>
+
+      <Card.Section>
+        <Skeleton count={1} data-testid="skills-loading" />
+      </Card.Section>
+
+      <Card.Footer className="bg-white border-0 pt-0 pb-2">
+        <Skeleton duration={0} data-testid="program-courses-count-loading" />
+      </Card.Footer>
+    </Card>
+  );
+
+  const programCard = (program) => {
+    const getProgramCourseCount = () => {
+      const numCourses = program.courseKeys?.length || 0;
+      if (!numCourses) {
+        return undefined;
+      }
+      return `${numCourses} ${numCourses > 1 ? 'Courses' : 'Course'}`;
+    };
+    const primaryPartnerLogo = getPrimaryPartnerLogo(partnerDetails[program.aggregationKey]);
+
+    return (
+      <Card isClickable>
+        <Card.ImageCap
+          src={program.cardImageUrl}
+          alt={program.title}
+          logoSrc={primaryPartnerLogo.src}
+          logoAlt={primaryPartnerLogo.alt}
+        />
+        <Card.Header
+          title={(
+            <Truncate lines={2} trimWhitespace>
+              {program.title}
+            </Truncate>
+          )}
+        />
+
+        <Card.Section className="py-3">
+          <>
+            {program.authoringOrganizations?.length > 0 && (
+              <p className="partner text-muted m-0">
+                <Truncate lines={1} trimWhitespace>
+                  {program.authoringOrganizations.map(org => org.key).join(', ')}
+                </Truncate>
+              </p>
+            )}
```

Comment:
[question] Are these authoring organizations technically the `Card.Header` `subtitle`?